api.md

```diff
@@ -1376,12 +1378,7 @@ one drop-chunks policy may exist per hypertable.
 | `if_not_exists` | (BOOLEAN) Set to true to avoid throwing an error if the policy already exists. A notice is issued instead. Defaults to false. |
 | `cascade_to_materializations` | (BOOLEAN) Set to `TRUE` to also remove chunk data from any associated continuous aggregates. Set to `FALSE` to only drop raw chunks (while keeping data in the continuous aggregates). Defaults to `NULL`, which errors if continuous aggregates exist.|
 
->:TIP: When dropping data from the raw hypertable while retaining data
-on a continuous aggregate, the `older_than` parameter to `drop_chunks`
-has to be longer than the `timescaledb.ignore_invalidation_older_than`
-parameter on the continuous aggregate. That is because we cannot
-process invalidations on data regions where the raw data has been
-dropped.
+>:TIP: When dropping data from the raw hypertable while retaining data on a continuous aggregate, the `older_than` parameter to `drop_chunks` has to be longer than the `timescaledb.ignore_invalidation_older_than` parameter on the continuous aggregate. That is because we cannot process invalidations on data regions where the raw data has been dropped.
```

Comment:
Can you split the paragraph in multiple lines as it was in origin? I think it is preferable approach except content placed in tables. Or was any specific reason to remove new lines for the paragraph?

Reply:
yes please split lines here

Reply:
It shows up very strange in my editor. Reverted the hunk.

Reply:
Also note that there are several very long lines elsewhere in the text, so I do not think keeping lines short can be a requirement or even a guideline.

Reply:
its not enforced anywhere but they should be kept short to make reviews easier